assets/js/amp-admin-tables.js

```diff
@@ -0,0 +1,46 @@
+/* global jQuery, ampAdminTables */
+( function( $ ) {
+	'use strict';
+
+	var adminTables = {
+
+		load: function() {
+			$( document ).ready( $.proxy( function() {
+				this.sourcesHideShow();
+				this.allSourcesHideShow();
+				this.addViewErrorsByTypeLinkButton();
+				this.boldURLInPageTitle();
+			}, this ) );
+		},
+
+		sourcesHideShow: function() {
+			$( 'span.dashicons.toggle-sources' ).on( 'click', function() {
+				$( this ).next( '.sources-container' ).toggleClass( 'collapsed' );
+				$( this ).toggleClass( 'dashicons-arrow-down dashicons-arrow-up' );
+			} );
+		},
+
+		allSourcesHideShow: function() {
+			$( '.double-arrow' ).on( 'click', function() {
+				$( '.double-arrow' ).find( '.dashicons' ).toggleClass( 'dashicons-arrow-down dashicons-arrow-up' );
+				$( 'span.dashicons.toggle-sources' ).toggleClass( 'dashicons-arrow-down dashicons-arrow-up' );
+				$( '.sources-container' ).toggleClass( 'collapsed' );
+			} );
+		},
+
+		addViewErrorsByTypeLinkButton: function() {
+			$( '.wp-heading-inline' ).after( '<a href="' + ampAdminTables.errorsByTypeLink + '" class="page-title-action">View errors by <strong>Type</strong></a>' );
```

Comment:
This shouldn't be using string concatenation for constructing HTML. It would be preferable to do `var link = $( '<a>...</a>' );` And then do `link.prop('href', ampAdminTables.errorsByTypeLink).`